### DISCUSSION TITLE
My power button is being mapped to the R1 button.

### DISCUSSION THREAD

**Discussion by Pakman450** [OnionUI/Onion#1605] [Support] [answered]
You heard that right. whenever I press the power button once, it makes the sound of the R button depending on the game. If I press and hold the power button to fully turn off the MMP, it goes turbo on the R button. 

how do I fix?

**Comment by Aemiii91** [comment, marked as answer]
Try unmapping R3/L3 in RetroArch core configuration

**Reply by Pakman450** [reply]
How do you unmap R3/L3?

**Reply by Pakman450** [reply]
I am in the retroarch->settings-> input, I am staring at the R3/L3 buttons, but I dont know how to get rid of it.

**Reply by Pakman450** [reply]
Now, my home button on the MMP is mapped to the L1 :(

**Reply by Pakman450** [reply]
nvm! I got it. I just had to map R3/L3 to R1/L1!!!

Thanks!

**Reply by Aemiii91** [reply]
Not sure that makes sense. But if you're happy, then I'm happy 😉
